source/amazon/services/prerequisites/dependencies.rst

```diff
@@ -64,4 +78,4 @@ To install boto3, execute the following command:
 
 .. code-block:: console
 
-  # pip3 install boto3==1.17.76
+  # pip install boto3==1.17.76
```

Comment:
We assumed that the machine could have different python versions above:

```shellsession
python3 get-pip.py
```

Why did you remove the explicit `pip3` call?

Reply:
This was restored from pip3 to pip as a mistake. It was set back to `pip3` instead.